src/org/thoughtcrime/securesms/ConversationFragment.java

```diff
@@ -549,8 +546,9 @@ public void releaseOutgoingMessage(long id) {
         }
     }
 
+    // Interface place holder
```

Comment:
placeholder*

Reply:
it was 2 am when I wrote that comment lol